### PR TITLE
Penultimations of the changelog kind

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
None

#### Purpose of change
Update the changelog for the month of Febuary
(Last PR said Febuary too, that one was a dirty liar)

#### Describe the solution
tools/generate_changelog.py -f --by-date changelog_02  -t ../repo_token -e 2021-02-01 2021-01-01
tac changelog_02 > changelog_02_rev
tools/merge_summaries.sh changelog_02_rev data/changelog.txt
Then copyedit the results.

#### Additional context
Typos happen, and some of the things I prune might be bigger than I remember or part of some larger set of changes that should go in the changelog, please take a look at not just the final diff, but also the copyedit commit to check for missed entries.

### CODE REVIEW

File: data/changelog.txt [hunk 1]
```diff
@@ -249,6 +258,21 @@ Makes NPCs able to fight back while guarding or patrolling.
 Fixes use-after-free error in sounds::process_sound_markers(player).
 Makes the Zombie technician's pull metal weapon attack drop items at the technician's feet.
 Fixes errors due to a bad memory access in drop_activity_actor::do_turn().
+Resolves a partial item spawn issue
+Prevents xp 'underflow' due to negative focus.
+Dragged vehicle/furniture no longer leaves a trail of memorized tiles in darkness.
+Prevents a crash by disallowing turning vehicles on ramps.
+Fixes crash when towing over a bridge.
+Fixes game freeze when player cannot carry a mission reward.
+Fixes crash in  `map::player_in_field` caused by iterator invalidation.
```
Comment (actual-nh) [inline data/changelog.txt:267]:
```suggestion
Fixes crashes from towing cables snapping or trying to stretch between z-levels.
Fixes a game freeze from happening when the player cannot carry a mission reward.
Fixes a crash in `map::player_in_field` caused by iterator invalidation.
```

**Review by actual-nh — COMMENTED** [review]
A first few - I need to check on some of the further-along ones, so wanted to go ahead and get these in.

File: data/changelog.txt [hunk 2]
```diff
@@ -274,6 +298,9 @@ Adds lightweight crafting inventory cache to improve NPC construction processing
 Reduces save size by skipping serialization of some item pockets and relic data.
 Adds RLE compression for JSON serialization/deserializations of data on items, reducing savefile size.
 Improves performance of initializing the achievements tracker.
+Only considers tiles with floor when NPC avoids explosives.
+Speeds up fungus processing by 30%+
```
Comment (actual-nh) [inline data/changelog.txt:302]:
```suggestion
When NPCs are fleeing from explosions, skips considering routing through tiles without a floor.
Speeds up fungus-related code processing by 30%+.
```

File: data/changelog.txt [hunk 3]
```diff
@@ -249,6 +258,21 @@ Makes NPCs able to fight back while guarding or patrolling.
...
+Fixes game freeze when player cannot carry a mission reward.
+Fixes crash in  `map::player_in_field` caused by iterator invalidation.
+Fixes a division by zero error when drinking water.
+Fixes NPC followers being blind after waking up.
+Fixes looks_like for vehicle parts.
+Fixed metabolic rehydration.
+Fixes L10N not working when env var LANG is unset.
+Avoids starvation when creating characters from older templates.
+Fixes moving furniture being 'destroyed' when grab-moving.
+Fixes crash when reloading container on the ground and canceling equipment.
```
Comment (actual-nh) [inline data/changelog.txt:275]:
```suggestion
Fixes Roots and Metabolic rehydration mutations so they actually relieve/eliminate the need for water.
Fixes localization not working if the LANG environmental variable is not set.
Avoids creating starving characters from older templates.
Fixes furniture being 'destroyed' when grab-moving.
Fixes crashes from canceling reloading a container that is on the ground.
```

File: data/changelog.txt [hunk 4]
```diff
@@ -84,6 +85,8 @@ Adds Hot Springs location.
 Adds a terrain flag 'NO_SHOOT' to prevent projectiles from striking or going through that tile type.
 Enables using a rope to carry large items.
 Adds large dermatik nest and tweaks dermatiks.
+New island prison layout.
+Adds the tazer drone.
```
Comment (actual-nh) [inline data/changelog.txt:89]:
```suggestion
Adds a new layout for the island prison challenge scenario.
Adds tazer drones.
```

File: data/changelog.txt [hunk 5]
```diff
@@ -360,6 +391,7 @@ Adds check enforcing almost-never-auto convention.
 Adds a GitHub workflow running flake8 to check the Python scripts.
 Consistently styles code in CMakeLists.txt files.
 Adds CMake linting.
+Captures ODR violations using GOLD linker, ASAN or LTO.
 
 ## I18N and A11Y:
 Fixes translations of martial art initiate and computer terminal messages.
```
Comment (actual-nh) [inline data/changelog.txt:397]: Shouldn't `Places cursor at highlighted item name in construction and crafting menus, as expected by screen readers.` be here, not under Bugfixes?

**Review by actual-nh — COMMENTED** [review]
OK, went through the rest.

File: data/changelog.txt (hunk 5 above)
Comment (kevingranade) [reply data/changelog.txt:397]: Seems reasonable, I'll move it.

File: data/changelog.txt [hunk 6]
```diff
@@ -249,6 +257,21 @@ Makes NPCs able to fight back while guarding or patrolling.
 Fixes use-after-free error in sounds::process_sound_markers(player).
 Makes the Zombie technician's pull metal weapon attack drop items at the technician's feet.
 Fixes errors due to a bad memory access in drop_activity_actor::do_turn().
+Resolves a partial item spawn issue
```
Comment (Night-Pryanik) [inline data/changelog.txt:260]:
```suggestion
Resolves a partial item spawn issue.
```

**Pull request merged**